site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -110,3 +115,30 @@ curl -GET "http://localhost:9358/gameserver" -H "accept: application/json"
 ```
 {"object_meta":{"creation_timestamp":"-62135596800","labels":{"agones.dev/sdk-foo":"bar"}},"spec":{"health":{}},"status":{"state":"Ready"}}
 ```
+
+## Running Local Mode in a Container
+
+Once you have your game server process in a container, you may also want to test the container build locally as well.
+
+Since the production agones-sdk binary has the `--local` mode built in, you can also use the production conatiner image
+locally as well!
+
+Since the SDK and your game server container need to share a port of `localhost`, one of the easiest ways to do that 
+is to have them both run with their networking occurring on the host, like so:
+
+In one shell run:
+
+```shell
+docker run --network=host --rm gcr.io/agones-images/agones-sdk:{{< release-version >}} --local
+```
+
+You should see a similar output to what you would if you were running the binary.
+
+Then in another shell, start your game server container
+
+```shell
+docker run --network=host --rm <your image here>
+```
+
+If you want to mount a custom `gameserver.yaml`, have a look at the 
```

Comment:
I think it would be useful to actually show the syntax here with an example path (like we do above) so that we don't force everyone who wants to do this to figure it out on their own. I haven't tried it, but I would think something like:

If you want to mount a custom `gameserver.yaml`, you will need to mount it into the agones-sdk container when running it:

```shell
mkdir /tmp/data
wget -P /tmp/data https://raw.githubusercontent.com/googleforgames/agones/main/examples/simple-game-server/gameserver.yaml
docker run --network=host --rm -v /tmp/data:/data gcr.io/agones-images/agones-sdk:1.25.0 --local -f /data/gameserver.yaml
```

For more information about how to specify mount paths, have a look at the [docker run](https://docs.docker.com/engine/reference/run/) documentation.

Reply:
You are right. Now I need to remember how to do it 😄